bin/run_epix

```diff
@@ -44,6 +44,11 @@ parser.add_argument('--Efield', dest='Efield',
 parser.add_argument('--MaxDelay', dest='MaxDelay', type=float,
                     action='store', default=1e7, #ns
                     help='Maximal time delay to first interaction which will be stored [ns]')
+parser.add_argument('--EventRate', dest='EventRate',
+                    action='store', default=-1, 
```

Comment:
I choose `-1` to avoid possible confusions with an `EventRate` of 0 Hz. But I have no strong opinion on that so if you prefer `0` for the clean separation we can change it.

Reply:
You can also put None as a default has the same effect and might be more clear.